man/systemd.timer.xml

```diff
@@ -51,7 +51,9 @@
     but simply left running. There is no concept of spawning new service instances in this case. Due to this, services
     with <varname>RemainAfterExit=</varname> set (which stay around continuously even after the service's main process
     exited) are usually not suitable for activation via repetitive timers, as they will only be activated once, and
-    then stay around forever.</para>
+    then stay around forever. Targets, which by default do not deactivate on their own, can be activated repeatedly by
+    timers by setting <varname>StopWhenUnneeded=true</varname> on the timer unit, so that it stops when all units
+    it depended on have deactivated and it is not depended on by any other unit itself.</para>
```

Comment:
Also, we usually say "depends on"/"is a dependency of". "be depended on" is harder to interpret.

Reply:
Fixed the typo, but I don't think "is a dependency of" expresses the same nuance in this context. The difference for me is about an active/passive distinction. Say you have a third unit loaded that depends on the target, so the target is a dependency of that unit, but said third unit is not running, so nothing "upholds" (to borrow otherwise overloaded terminology) the target at this moment. This would be more correctly expressed with a progressive "it is not being depended on", but that sounds clanky to me.